random/random.go

```diff
@@ -148,26 +163,26 @@ func ParseDistribution(s string) (Distribution, error) {
 	case "fixed":
 		n, err := strconv.ParseUint(val, 10, 32)
 		if err != nil {
-			return nil, errInvalid
+			return nil, errors.New(fmt.Sprintf("value for fixed distribution is invalid: %s", err))
 		}
 		return &Fixed{
 			Value: int64(n),
 		}, nil
 	case "uniform":
 		p := strings.Split(s[i+1:j], "..")
 		if len(p) != 2 {
-			return nil, errInvalid
+			return nil, errors.New("interval for uniform distribution has invalid format, expected: min..max")
 		}
 		min, err := strconv.ParseUint(p[0], 10, 32)
 		if err != nil {
-			return nil, errInvalid
+			return nil, errors.New(fmt.Sprintf("min parameter uniform distribution is invalid: %s", err))
```

Comment:
errors.New(fmt.Sprintf(...)) => fmt.Errorf(...)

Reply:
Cool, didn't know about that.

Reply:
My 2c I'd love to see https://github.com/pkg/errors instead of strings in this project.

Reply:
Ok, I will use this package.